### PR TITLE
Update validation loss and links for Jorge Vanco

### PR DESCRIPTION
**Pull request by JorgeVanco**
I reduced the number of layers to 12 and doubled the batch size to 256. Had to modify the number of steps to 18200 steps.

### CODE REVIEW

**Comment by JorgeVanco** [comment]
​I added learned value embeddings just like in the modded nano-gpt repo. I have also further reduced the number of layers to 10.

​I have also added de muon momentum warmup. It barely has any improvement, but as it is already very hard to improve, I might as well add it as it does not affect the speed.

**Comment by vskogstad** [comment]
Thats amazing improvement! 👍 

When evaluating on the validation set, I assume you do so with the new maximum context length(1792)? 
I noticed for my model that validation loss decreases by just doubling the context length and decreasing batch size during validation. Even with no training at extended context. This is/was also done in the NanoGPT-speedrun at some point. It seems like we get some gains during validation just from decreasing the occurences when the model has very little available context and has to make a guess. 
(Just to be clear: I think you validating your model with large context length is correct, as you've actually trained the model on that context length.)

**Comment by JorgeVanco** [comment]
Thank you! 
Yes exactly, I am running validation with the maximum context length.

**Comment by marcelroed** [comment]
Ah, I should have made this clear in this repo earlier (will add it to the readme) but verification will happen at context length 512. I think this might have been something we communicated over our class Slack which should have been added to the writeup.

Could you eval using these settings? Sorry for the confusion!

**Comment by marcelroed** [comment]
Great work!

**Comment by marcelroed** [comment]
Sorry for the delayed response @vskogstad! It is measured as accumulated loss across batches of 512. Let me add some clarifications in the README, but to answer your questions:

Everything is meant to be from scratch, so you may _not_ use any of `torch.nn`, SDPA or Flex Attention. You can write your own kernels, but in order for verification to be possible, we have to limit you to Triton (no CUDA). In the next iteration of the class we'll likely use some CUDA Python.

**Comment by marcelroed** [comment]
For next iteration (this spring) I will also add a verification example with a different seed from the one I use on my end so it'll be easier to get a representative measurement for the leaderboard. It's a bit confusing when the testing framework and thus scores are also built by each student individually.
